starknet/tests/eth-sig-auth.test.ts

```diff
@@ -0,0 +1,258 @@
+import fs from 'fs';
+import { ethers } from 'ethers';
+import { Provider, Account, CallData, cairo, json } from 'starknet';
+import {
+  Propose,
+  proposeTypes,
+  voteTypes,
+  Vote,
+  updateProposalTypes,
+  UpdateProposal,
+} from './types';
+
+const pk = process.env.PRIVATE_KEY || '';
+
+describe('Starknet Signature Authenticator', () => {
+  const signer = new ethers.Wallet(pk);
+  const provider = new Provider({ sequencer: { baseUrl: 'http://127.0.0.1:5050' } });
+  // devnet predeployed account
+  const privateKey0 = '0xe3e70682c2094cac629f6fbed82c07cd';
+  const address0 = '0x7e00d496e324876bbc8531f2d9a82bf154d1a04a50218ee74cdd372f75a551a';
```

Comment:
maybe better off as constants so we can use them in other test files ?

Reply:
btw I think we're better off querying them programatically. They will change at each `starknet-devnet` restart (unless `--seed` is given)

Reply:
dont want to spend to long on make this all work. else im just re-inventing starknet hardhat plugin

Reply:
gotcha, then I feel like using a `seed` will definitely help for reproducing the tests :)

Reply:
added a comment